odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -4085,3 +4085,67 @@ def test_precompute_monetary(self):
         ]
         with self.assertQueries(QUERIES):
             model.create({})
+
+
+class TestModifiedPerformance(common.TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Modified = cls.env['test_new_api.modified']
+        cls.ModifiedLine = cls.env['test_new_api.modified.line']
+        cls.modified_a = cls.Modified.create({
+            'name': 'Test',
+        })
+        cls.modified_line_a = cls.ModifiedLine.create({
+            'modified_id': cls.modified_a.id,
+            'quantity': 5,
+            'price': 1,
+        })
+        cls.modified_line_a_child = cls.ModifiedLine.create({
+            'modified_id': cls.modified_a.id,
+            'quantity': 5,
+            'price': 2,
+            'parent_id': cls.modified_line_a.id,
+        })
+        cls.modified_line_a_child_child = cls.ModifiedLine.create({
+            'modified_id': cls.modified_a.id,
+            'quantity': 5,
+            'price': 3,
+            'parent_id': cls.modified_line_a_child.id,
+        })
+        cls.env.invalidate_all()  # Clean the cache
+
+    def test_modified_trigger_related(self):
+        with self.assertQueries([], flush=False):
```

Comment:
Just a matter of taste, but isn't this more readable?
```suggestion
        with self.assertQueryCount(0, flush=False):
```
The same suggestion applies to all cases below.